openy_af4_vue_app.md

```diff
@@ -1,33 +1,23 @@
 # README
 
-## Vue CLI
-
-Installation `npm install -g @vue/cli`
-
-Check version `vue --version` =&gt; @vue/cli 4.0.5
-
```

Comment:
We don't need Vue CLI to build the application. It was misunderstanding from my side.